include/menu_option.h

```diff
@@ -15,7 +15,7 @@ using namespace std;
  **/
 class MenuOption {
 
-    private:
+    protected:
```

Comment:
This is to make the member variables of "MenuOption" available for its daughter-classes, but not available outside of this scope.